src/lwc/geGiftEntryFormApp/geGiftEntryFormApp.js

```diff
@@ -253,7 +254,7 @@ export default class GeGiftEntryFormApp extends NavigationMixin(LightningElement
         recordId: '$recordId',
         fields: [
             BATCH_ID_FIELD,
-            BATCH_NAME
+            BATCH_NAME,
```

Comment:
This is really minor but the comma at the end of line 257 should be removed

```suggestion
            BATCH_NAME
```